lighthouse-core/test/results/sample_v2.json

```diff
@@ -1404,7 +1403,7 @@
           {
             "resourceType": "image",
             "label": "Image",
-            "requestCount": 2,
+            "requestCount": 1,
```

Comment:
The image & total request counts changed because the sample page includes an image blob & blobs are now ignored.

Reply:
gotcha! hurray fixes 🎉